configuration/klippy/beacon_mesh.py

```
@@ -33,142 +116,822 @@ def register_handler(self):
+		contact_mesh_points = self.bed_mesh.pmgr.get_profiles()[contact_mesh_name]["points"][:]		
+		contact_params = self.bed_mesh.z_mesh.get_mesh_params()
+		contact_x_step = ((contact_params["max_x"] - contact_params["min_x"]) / (contact_params["x_count"] - 1))
+		contact_y_step = ((contact_params["max_y"] - contact_params["min_y"]) / (contact_params["y_count"] - 1))
+
+		self.ratos.debug_echo("Create compensation mesh", "Filtering contact mesh")
+		contact_mesh_points = self._apply_filter(contact_mesh_points)
+		contact_params[RATOS_MESH_NOTES_PARAMETER] = "contact mesh filtered using local low filter"
+
+		compensation_mesh_points = []
+		
+		eventtime = self.reactor.monotonic()
```

Ruff (0.12.2)

Local variable `eventtime` is assigned to but never used

Remove assignment to unused variable `eventtime`

(F841)